src/routes/friend.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const router = express.Router();
+const User = require("../databases/schema/users");
+
+//Send a friend request to a user and add the user to the senders list of pending requests
+//Incoming: user's object _id, username to search for
+//Outgoing: status 200/error will be empty
+router.put("/addFriend", async (request, response) => {
```

Comment:
To keep the api consistent, maybe go with `/add` (full: `/friend/add`)?

Reply:
Add is ambiguous. What am I adding? Am I adding attributes? There's no other add functions, so there's really no consistency gained by changing this. It's better to make sure someone knows exactly what they're doing when they ping the endpoint.

Reply:
But isn't `/friend/addfriend/` repetitive?

Reply:
Repetition is better than ambiguity